core/tests/protractor_desktop/topicsAndSkillsDashboard.js

```diff
@@ -0,0 +1,91 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview End-to-end tests for the topics and skills dashboard page.
+ */
+
+var forms = require('../protractor_utils/forms.js');
+var general = require('../protractor_utils/general.js');
+var users = require('../protractor_utils/users.js');
+var waitFor = require('../protractor_utils/waitFor.js');
+var workflow = require('../protractor_utils/workflow.js');
+
+var AdminPage = require('../protractor_utils/AdminPage.js');
+var TopicsAndSkillsDashboardPage =
+  require('../protractor_utils/TopicsAndSkillsDashboardPage.js');
+
+describe('Topics and skills dashboard functionality', function() {
```

Comment:
Can you give us a list of functional capabilities for the dashboard? In general, let us try to come up with that list first, and then write the tests (so we know we have 100% coverage).

Reply:
I have added the paths that I have used, and those that are pending in the Topics and Skills Dashboard to the functional capabilities doc.